src/PersonalSite.WebApi/Program.cs

```diff
@@ -37,6 +38,7 @@
 builder.Services.AddSingleton<IClock, Clock>();
 
 builder.Services.AddMediatR(typeof(PersonalSite.WebApi.IAssemblyMarker));
```

Comment:
Can we remove that line? I think there are no request handlers in the webapi atm, maybe we don't need that anymore